doc/caches.rst

```diff
@@ -89,6 +89,11 @@ This is the default cache type and it uses a single file for each tile. Availabl
 
   .. versionadded:: 2.0.0
 
+``directory_permissions``, ``file_permissions``:
+  Permissions that MapProxy will set when creating files and directories. Must given as strings containing the octal representation of permissions. I.e. ``rwxrw-r--`` is ``'764'``.
```

Comment:
```suggestion
  Permissions that MapProxy will set when creating files and directories. Must be given as string containing the octal representation of permissions. I.e. ``rwxrw-r--`` is ``'764'``. This will not work on windows OS.
```

Reply:
good point